modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -528,7 +528,13 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
       const allOutputs = [...explanation.outputs, ...explanation.changeOutputs];
 
       // verify that each recipient from txParams has their own output
-      const expectedOutputs = _.get(txParams, 'recipients', [] as TransactionRecipient[]);
+      const expectedOutputs = _.get(txParams, 'recipients', [] as TransactionRecipient[]).map((output) => {
+        if (!output.address.startsWith('bitcoincash')) {
+          return output;
+        }
```

Comment:
You can remove this. The `canonicalAddress` will work on any bch address.

Reply:
yes, updated this one too.